### DISCUSSION TITLE
SelexION Sciex 5600+ swath data

### DISCUSSION THREAD

**Discussion by GeorgeCMarinescu** [vdemichev/DiaNN#584] [General]
Hi,

What do you think will be the performance impact of SelexION  in terms of proteins identifications number by dia-nn for a complex sample, on a Sciex 5600+ swath data, micro flow ESI. We usually have between 3000 and 5000 gene level ids for a whole cell lysate protein extract.
What do you estimate will be the numbers if we upgrade this instrument with selexION ?

Regards,
George

**Comment by vdemichev** [comment]
Hi George,

DIA-NN would not support it, special algorithms would be required to make it work.

Best,
Vadim

**Comment by GeorgeCMarinescu** [comment]
Thanks for the quick reply !
